docs/HowToFAQ/FAQSingleton.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How do I make a singleton instance of a class for repeated later use?
+---
+
+## Question
+
+How do I make a singleton instance of a class for repeated later use?
+In general, one might need to allocate an object, do some initialization,
+but then treat it as const from then on.
```

Comment:
Is that really called a singleton?

Reply:
I think so. In this cases it is the language the questioner used. I suppose one might make a distinction between a class of which a program only ever allocates a single instance (as here) vs. a class in which every subsequent attempt to allocate a class instance always returns the same original instance (perhaps the technical definition of the pattern).

I didn't think the distinction was worth berating here.